commands/deploy_command.go

```diff
@@ -226,7 +226,7 @@ func (c *DeployCommand) Execute(args []string) ExecutionStatus {
 	}
 
 	// Print initial message
-	ui.Say("Deploying multi-target app archive %s in org %s / space %s as %s...",
+	ui.Say("Deploying multi-target app archive %s in org %s / space %s as %s...\n",
```

Comment:
I think that ui.Say is printing \n by default?

Reply:
Yeah, but I need another one. :D

Otherwise we get:
```
Deploying multi-target app archive C:\Users\i318495\Desktop\mtas\portal-cf-test-content.mtar in org deploy-service / space i318495 as I318495...
Attention: You've specified a custom chunk size (5 MB) via the environment variable "CHUNK_SIZE_IN_MB".
Uploading 1 files...
  C:\Users\i318495\Desktop\mtas\portal-cf-test-content.mtar
OK
```
As opposed to:
```
Deploying multi-target app archive C:\Users\i318495\Desktop\mtas\portal-cf-test-content.mtar in org deploy-service / space i318495 as I318495...

Attention: You've specified a custom chunk size (5 MB) via the environment variable "CHUNK_SIZE_IN_MB".
Uploading 1 files...
  C:\Users\i318495\Desktop\mtas\portal-cf-test-content.mtar
OK
```